.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -13,6 +13,10 @@ labels: 'feature'
 <!-- A clear and concise description of the feature you would like to add, and how it solves the motivating problem. -->
 
 
+### Screenshot
+<!-- Screenshot of the part which needs be changed -->
+
+
```

Comment:
How does this work? 🙈 I can see it for some features, but would this not mostly be screenshots of "this is how the CMS currently looks. Now imagine if there was another feature here, even though it is not"? 🙈

Reply:
Ah, OK! Got it!